pandas/core/ops.py

```diff
@@ -513,14 +513,14 @@ def _get_op_name(op, special):
 other : Series, DataFrame, or constant
 axis : {{0, 1, 'index', 'columns'}}
     For Series input, axis to match Series index on
+level : int or name
```

Comment:
I think `level : int or str`, though we're inconsistent here.

@jorisvandenbossche what should our policy be on index positions or labels? In principle, `int or str` isn't quite right since index labels needn't be strings. It'd be `int or object`, which isn't that informative. `int or label`?

Reply:
Hmm, yes, this is a dubious one. "name" is technically not a 'type', but "str" would not be correct as you say. So maybe better to keep it that way. 

If we keep the 'name' or 'label', wouldn't 'name' be better because it is here a index level name, not a label inside the index ?

Reply:
Ah, yes you're right I was mistaken.

And we can declare that "name" means anything that can go in `{.index,.columns}{.name,.names}`.